src/_about/newcomer-guide.md

```diff
@@ -0,0 +1,370 @@
+* Acting as a form of oversight over the rest of the project, ensuring that processes are followed as documented and
+  aiding with internal conflicts
+* Acting as tie-breakers in Quilt's internal democratic processes, voting on issues that are in stalemates to get them
+  moving along again
+* Managing The Quilt Project's finances, and
+  [ensuring that income and outgoings are documented transparently](https://opencollective.com/quiltmc)
+* Handing out access permissions to the rest of the teams as needed, and acting as an ownership group on platforms that
+  require it and aren't managed by the Community Team -- such as [on GitHub](https://github.com/QuiltMC)
+* Aiding with the decision-making process by providing their expertise to the other teams, and taking charge on tasks
+  that don't fall under the jurisdiction of another team
+
+Despite its name, the Admin Board does not hold ultimate power over the rest of the project or its staff -- the other
+teams operate independently, and the Admin Board acts as a supporting group for the rest of those teams.
+
+### The Community Team
+
+The Community Team is a specialist team that oversees all of Quilt's community spaces, as well as its social media
+accounts and website, and handling any public relations tasks that come up. It's further subdivided into a number of
+separate roles, in order to better distribute work and prevent abuses of access privileges.
+
+#### Keyholders
+
+The title of Keyholder refers to an individual that is appointed via an internal democratic election, and who exists to
+hold the position of "Owner" in all official Quilt community spaces that do not have a reasonable concept of group
+ownership.
+
+Keyholders are reputable, well-known members of the wider Minecraft modding community that otherwise have no interest
+in Quilt, but who would personally suffer a large blow to their reputation if they were to abuse their position.
+Because of this, we hope that we can avoid any potential abuses of power.
+
+A Keyholder will oversee the Community Team when required, help to ensure that Quilt's policies are followed in
+internal Community Team spaces, and hand out permissions to the rest of the Community Team as required. They're also
+welcome to weigh in on anything going on internally, but they're otherwise uninvolved with the project.
+
+#### Community Managers
+
+Community Managers oversee the daily operations of Quilt's official community spaces. As well as supporting the rest
+of the Community Team and acting as Moderators, they handle administrative tasks in the community spaces, lead
+interviews for Community Team applications, maintain and keep track of the tools used to keep the community spaces
+running smoothly, and create and update Community Team policies.
+
+Additionally, Community Managers lead investigations into wider social trends and controversies in the Minecraft
```

Comment:
Is this really a job of managers specifically, and not moderators?

Reply:
I can't really think of an example where this has been entirely moderator-lead, yeah. It makes sense for managers to oversee this.

Reply:
Fair enough.

If the RFC doesn't reflect this, it probably should.